docs/vue/your-first-app.md

```diff
@@ -146,30 +161,53 @@ Open `/src/views/Tab2.vue`. We see:
     </ion-content>
   </ion-page>
 </template>
+
+<script setup lang="ts">
+import { IonPage, IonHeader, IonToolbar, IonTitle, IonContent } from '@ionic/vue';
+import ExploreContainer from '@/components/ExploreContainer.vue';
+</script>
 ```
 
-`ion-header` represents the top navigation and toolbar, with "Tab 2" as the title. Let’s rename it:
+`ion-header` represents the top navigation and toolbar, with "Tab 2" as the title (there are two of them due to iOS [Collapsible Large Title](../api/title.md#collapsible-large-titles) support). Rename both `ion-title` elements to:
 
-```html
+```vue
 <ion-title>Photo Gallery</ion-title>
```

Comment:
This may be confusing if people interpret it to change the large title to this:

```diff
-<ion-title size="large">Tab 2</ion-title>
+<ion-title>Photo Gallery</ion-title>
```

I realize this is the same as the Angular example, just pointing it out because I didn't notice it until this review.

Reply:
Made the change for Angular: https://github.com/ionic-team/ionic-docs/pull/4157/commits/c19dbea64e542db250c18deeb74c1b76e707da06

Reply:
https://github.com/ionic-team/ionic-docs/pull/4197/commits/5ba929275907d3b20cdc9ec9c514feba65d3bf84